app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/ChatViewHolder.java

```diff
@@ -229,8 +229,11 @@ private void rateSusiSkill(final String polarity, String locationUrl, final Cont
         if (susiLocation.size() == 0)
             return;
 
-        Call<SkillRatingResponse> call = new ClientBuilder().getSusiApi().rateSkill(susiLocation.get("model"),
-                susiLocation.get("group"), susiLocation.get("language"), susiLocation.get("skill"), polarity);
+        SkillRatingQuery queryObject = new SkillRatingQuery(susiLocation.get("model"), susiLocation.get("group"),
+                susiLocation.get("language"), susiLocation.get("skill"), polarity);
+        QueryHelper queryHelper = new QueryHelper();
```

Comment:
Why is there a need to instantiate this?

This is basically a static method, besides the functions in this class should be placed in client builder